packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -430,6 +430,69 @@ abstract class ScrollView extends StatelessWidget {
 /// ```
 /// {@end-tool}
 ///
+/// {@tool dartpad --template=stateful_widget_material}
+///
+/// This sample code shows a scroll view that contains two [SliverList]s,
```

Comment:
Need to start out by explaining why we're setting the scroll view up in this way.

Reply:
I updated the description now, to try to explain the use case.